project/plugins.sbt

```diff
@@ -6,16 +6,17 @@ resolvers ++= Seq(
   "Twitter Maven" at "http://maven.twttr.com"
 )
 
-addSbtPlugin("com.eed3si9n"       % "sbt-assembly"        % "0.10.2")
-addSbtPlugin("com.eed3si9n"       % "sbt-unidoc"          % "0.3.3")
-addSbtPlugin("com.fortysevendeg"  % "sbt-microsites"      % "0.3.3")
-addSbtPlugin("com.github.gseitz"  % "sbt-release"         % "1.0.0")
-addSbtPlugin("com.jsuereth"       % "sbt-pgp"             % "1.0.0")
-addSbtPlugin("com.twitter"        %% "scrooge-sbt-plugin" % "4.14.0")
-addSbtPlugin("com.typesafe"       % "sbt-mima-plugin"     % "0.1.14")
-addSbtPlugin("com.typesafe.sbt"   % "sbt-ghpages"         % "0.5.4")
-addSbtPlugin("com.typesafe.sbt"   % "sbt-git"             % "0.6.2")
-addSbtPlugin("com.typesafe.sbt"   % "sbt-scalariform"     % "1.3.0")
-addSbtPlugin("org.scoverage"      % "sbt-scoverage"       % "1.5.0")
-addSbtPlugin("org.xerial.sbt"     % "sbt-sonatype"        % "1.0")
-addSbtPlugin("org.wartremover"    % "sbt-wartremover"     % "2.1.1")
+addSbtPlugin("com.eed3si9n"             % "sbt-assembly"        % "0.10.2")
+addSbtPlugin("com.eed3si9n"             % "sbt-unidoc"          % "0.3.3")
+addSbtPlugin("com.fortysevendeg"        % "sbt-microsites"      % "0.3.3")
+addSbtPlugin("com.github.gseitz"        % "sbt-release"         % "1.0.0")
+addSbtPlugin("com.jsuereth"             % "sbt-pgp"             % "1.0.0")
+addSbtPlugin("com.twitter"              %% "scrooge-sbt-plugin" % "4.14.0")
+addSbtPlugin("com.typesafe"             % "sbt-mima-plugin"     % "0.1.14")
+addSbtPlugin("com.typesafe.sbt"         % "sbt-ghpages"         % "0.5.4")
+addSbtPlugin("com.typesafe.sbt"         % "sbt-git"             % "0.6.2")
+addSbtPlugin("com.typesafe.sbt"         % "sbt-scalariform"     % "1.3.0")
+addSbtPlugin("org.scoverage"            % "sbt-scoverage"       % "1.5.0")
+addSbtPlugin("org.xerial.sbt"           % "sbt-sonatype"        % "1.0")
+addSbtPlugin("org.wartremover"          % "sbt-wartremover"     % "2.1.1")
+addSbtPlugin("com.typesafe.sbteclipse"  % "sbteclipse-plugin"   % "5.2.3")
```

Comment:
do we need to add this, or can you put it in your personal `~/.sbt` directory?

Reply:
it's convenient for me because I work on projects that require different versions of the plugin so it can't be on `~/.sbt`. It's fine, I'll revert this change.